### ISSUE TITLE
Tab-Navigation

### ISSUE THREAD

**Issue by naweidner** [elan-ev/tobira#654]
Tab-Navigation is important for users who are unable to use a mouse (or prefer not to use one) to properly navigate through a web application. Two main things I've noticed: 

**1.** The **tab order** is not always correct and sometimes confusing. In some places, the focus doesn't shift where you expect it to. 
My Firefox accessibility tab (right click > inspect > accessibility) and my Wave extension show the correct tab order. So I'm not exactly sure what's causing this.

**Example:** 
When I navigate to "Speakers" (and open this view/page with enter) and press Tab again, the focus isn't where I expected it to be. I would expect the focus to be on "Home"  ("Startseite") in the navigation, or back to the logo "Opencast University", or the breadcrumbs' home icon. 

Instead the focus is: 
- on "D-BSSE: Lunch Meetings Molecular Systems Engineering" (see screenshot) [Firefox 108 (Win 11) & FF 107 (Fedora 35)] 
- or on "D-ARCH" [Chrome 108 (Win 11 & Fedora 35)] 

![GetImage](https://user-images.githubusercontent.com/93541078/211565787-d241c927-305e-4a8e-9cab-2ebabe57fbb7.png)

**2.** The **focus-border** is very thin and not consistent everywhere, there are a few elements that have a different border-color and border-with. The color contrast is about 4.5 which is good, but I would make the border stand out more, maybe a bit thicker. 

**Example:** 
The **buttons** have a different border-color and border-width than other elements (like the navigation, see screenshot above). The border is thick and grey. Exception is the login button, here the border-color is black (in Firefox and Chrome)

Depending on the Browser, **links** have a different border-color. Links are outlined black in Chrome and blue (a different blue like the navigation has) in Firefox (see last two pictures). 

![GetImage(1)](https://user-images.githubusercontent.com/93541078/211566058-05b1bee0-3873-4b58-b13a-92608e33f329.png)
![grafik](https://user-images.githubusercontent.com/93541078/211571270-f131b405-1b1d-4668-887c-3c80c7e05ad9.png)

Focus border in Chrome:
![GetImage(2)](https://user-images.githubusercontent.com/93541078/211566089-2f495fb6-b8d2-4355-b3b0-572908e847a8.png)

Focus border in Firefox:
![GetImage(3)](https://user-images.githubusercontent.com/93541078/211566107-9e9926e2-2b0b-407f-9261-4b2514c9c148.png)

**Comment by owi92** [comment]
The tab order appears to have been fixed, I believe with #680.
